### ISSUE TITLE
ModernFlyouts support

### ISSUE THREAD

**Issue by Movisivi** [Taiko2k/TauonMusicBox#922]
Can you add for this? https://github.com/ModernFlyouts-Community/ModernFlyouts

**Comment by Taiko2k** [comment]
I agree this would be nice to have. Ill try aim to have support in Tauon added at some point.

**Comment by Taiko2k** [comment]
Ive been digging into this and it seems quite obtuse, especially trying to do it from outside of a UWP app. Basically you have to use the Windows Runtime API and you need to do some whacko nonsense to be able to access it from a desktop app, but nothing about that is simple. And even then there does seem to be bugs (as mentioned in the discussion https://github.com/microsoft/WindowsAppSDK/issues/127)

But other apps like Firefox do do it, so surely it's possible. But I think I might need a more seasoned Windows developer to help me with this. Like someone who knows a COM object is.

**Comment by Movisivi** [comment]
Hope you get it figured out. I've read that Microsoft documentation isn't all that great.  I'm just curious,  which would be easier/take less time to do: make Tauon a UWP app or add support for this feature as Tauon is?

**Comment by Taiko2k** [comment]
OK major progress. Added support in Tauon and It works!

Unfortunately I still had trouble with getting album art thumbnails to work. Seek bar and click to focus also aren't working. But basic metadata and play/pause/next/previous buttons are working.

![Screenshot 2023-02-12 220603](https://user-images.githubusercontent.com/17271572/218302219-f8ca1209-79a9-4bc1-8f75-7f1bdff1b0af.png)


Ill do a release with this new feature sometime soon. Further improvements will have to come at a future date as Ive sunk too much time into it for now.

**Comment by Movisivi** [comment]
I saw your commits, I can't imagine how long that took you lol. I'm grateful for it, I will eagerly look forward to this feature whenever I do a random winget upgrade
